lib/util/include/hse_util/compiler.h

```diff
@@ -8,95 +8,114 @@
 
 #include "build_config.h"
 
+/* clang-format off */
+
 /* Ubuntu 18.04 uses glibc 2.27, and threads.h only exists in glibc >= 2.28 */
 #if defined(__has_include) && !__has_include(<threads.h>)
```

Comment:
I am thinking removing this check/define and moving it into Meson would be better. I hate having it here.

Reply:
I'll take a look in my next PR if you want...